include/openssl/store.h

```diff
@@ -191,7 +191,8 @@ void OSSL_STORE_SEARCH_free(OSSL_STORE_SEARCH *search);
 
 /* Search term accessors */
 int OSSL_STORE_SEARCH_get_type(const OSSL_STORE_SEARCH *criterion);
-X509_NAME *OSSL_STORE_SEARCH_get0_name(const OSSL_STORE_SEARCH *criterion);
+const X509_NAME *OSSL_STORE_SEARCH_get0_name(const OSSL_STORE_SEARCH
+                                             *criterion);
```

Comment:
Don't change the return type of existing public functions.

Reply:
Well, I know that this can cause compatibility issues, but maybe this is acceptable for v3.0?

Reply:
3.0 is trying to limit the amount of breaking changes..